sdks/java/io/rabbitmq/build.gradle

```diff
@@ -28,8 +28,9 @@ dependencies {
   compile library.java.joda_time
   compile "com.rabbitmq:amqp-client:4.9.3"
   testCompile project(path: ":sdks:java:io:common", configuration: "testRuntime")
-  testCompile "org.apache.qpid:qpid-broker:0.28"
-  testCompile "org.apache.qpid:qpid-broker-core:0.28"
+  testCompile "org.apache.qpid:qpid-broker-core:7.1.5"
```

Comment:
Upgrading QPid. The previous version was from 2013. Yikes.

Reply:
2013?  That's only...wait, _six_ years ago?